### ISSUE TITLE
ModuleNotFoundError: scalar_formula_request_queries

### ISSUE THREAD

**Issue by gusantos1** [DataDog/datadog-api-client-python#1496]
This morning I started getting this error in my workload.

pip install datadog-api-client

from datadog_api_client.v2.model.scalar_formula_request_queries import ScalarFormulaRequestQueries
**ModuleNotFoundError**: No module named 'datadog_api_client.v2.model.**scalar_formula_request_queries**'

**Comment by therve** [comment]
Hi, this was removed in #1431 as it was blocking a few other issues,. You should be able to just remove its use without others side effects.
Let's us know if that works for you, thanks.

**Comment by gusantos1** [comment]
I used it in queries on ScalarFormulaQueryRequest. Now I don't know which class to instantiate
queries=ScalarFormulaRequestQueries(
                    [
                        MetricsScalarQuery(
                            aggregator=MetricsAggregator.SUM,
                            data_source=MetricsDataSource.METRICS,
                            query=...,
                            name=...
                        ),
                    ]
                ),

**Comment by chamini2** [comment]
From the same PR we are now getting

```
ModuleNotFoundError: No module named ‘datadog_api_client.v2.model.http_log’
```

I think this should have been a major version bump. It is backwards incompatible and it's breaking packages using datadog_api_client.

**Comment by therve** [comment]
> I used it in queries on ScalarFormulaQueryRequest. Now I don't know which class to instantiate 

Don't instantiate any class, just pass `queries=[]`.

I'm sorry it's breaking your code. We made the change thinking it was an improvement and that the migration should be fairly trivial.

**Comment by gusantos1** [comment]
Temporarily my solution was to go back to 2.12. I will study better how to adapt. thanks for the quick help

**Comment by skarimo** [comment]
Thanks for the report. We have released [`v2.13.1`](https://github.com/DataDog/datadog-api-client-python/releases/tag/2.13.1) of the client which should resolve this. Feel free to re-open if the issue persists. Thanks again!
